governance/votes/Executive Vote - February 22, 2023.md

```diff
@@ -0,0 +1,104 @@
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. This is to ensure that at least some auction keepers are available to react to unforeseen issues with their bots in the event of early liquidations on new collateral types.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Aave v2 DAI Direct Deposit Module (D3M) Onboarding
+
+As per this [poll](https://vote.makerdao.com/polling/QmUMyywc#poll-detail), the Aave v2 D3M will be onboarded with the following parameters, if this executive proposal passes. 
+
+#### Vault Parameters
+
+* [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line): **5 million DAI**.
+* [Target Available Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap): **5 million DAI**.
+* [Ceiling Increase Cooldown (`ttl`)](https://manual.makerdao.com/module-index/module-dciam#ceiling-increase-cooldown-ttl): **12 hours**.
+* [Target Borrow Rate (`bar`)](https://manual.makerdao.com/module-index/module-dai-direct-deposit#target-borrow-rate-bar): **2%**.
+
+Note that Governance has approved a `line` of 20 million DAI but the intial deployment sets the `line` to 5 million DAI following the advice of the Protocol Engineering Core Unit. 
+
+#### Threads and Assessments 
+
+Please review the following resources containing information about the Aave V2 D3M to inform your position before voting.
+* [Proposal Thread](https://forum.makerdao.com/t/parameter-changes-proposal-ppg-omc-001-2-february-2023/19637)
+* [Initial Risk Core Unit Evaluation](https://forum.makerdao.com/t/d3m-risk-assessment/9353)
+* [Initial Protocol Engineering Core Unit Evaluation](https://forum.makerdao.com/t/direct-aavev2-dai-direct-deposit-module-technical-assessment/10751)
+* [MIP 50: DAI Direct Deposit Module](https://mips.makerdao.com/mips/details/MIP50)
+
+
+### Strategic Finance Core Unit (SF-001) MKR Transfer
+
+As per their successful [MKR vesting proposal](https://vote.makerdao.com/polling/QmZn66qt), the following MKR streams will be added to the [Strategic Finance Core Unit (SF-001)](https://mips.makerdao.com/mips/details/MIP39c2SP36) starting March 1, 2022 through March 1, 2025 in accordance with their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP48), if this executive passes. The MKR is claimable on March 1, 2023.
+- **240 MKR** will be streamed to [0x31C01e90Edcf8602C1A18B2aE4e5A72D8DCE76bD](https://etherscan.io/address/0x31C01e90Edcf8602C1A18B2aE4e5A72D8DCE76bD). 
+- **195 MKR** will be streamed to [0x12b19C5857CF92AaE5e5e5ADc6350e25e4C902e9](https://etherscan.io/address/0x12b19C5857CF92AaE5e5e5ADc6350e25e4C902e9)
```

Comment:
Improve readability

```suggestion
As per their successful [MKR vesting proposal](https://mips.makerdao.com/mips/details/MIP40c3SP48), the following MKR streams will be added for the [Strategic Finance Core Unit (SF-001)], if this executive proposal passes. 

#### Contributor Stream 1

- Amount to be Streamed: **240 MKR**
- Start Date: **2022-03-01**
- End Date: **2025-03-01**
- Cliff Date: **2023-03-01**
- Destination Address: [0x31C01e90Edcf8602C1A18B2aE4e5A72D8DCE76bD](https://etherscan.io/address/0x31C01e90Edcf8602C1A18B2aE4e5A72D8DCE76bD)

#### Contributor Stream 2

- Amount to be Streamed: **195 MKR**
- Start Date: **2023-03-01**
- End Date: **2025-03-01**
- Cliff Date: **2023-03-01**
- Destination Address: **[0x12b19C5857CF92AaE5e5e5ADc6350e25e4C902e9](https://etherscan.io/address/0x12b19C5857CF92AaE5e5e5ADc6350e25e4C902e9)**
```

Reply:
Just double checking that this isn't a typo: Contributor 2's stream start date is also 2022 in the spreadsheet

Reply:
Also, wasn't sure if you meant to delete the link in this part "MKR streams will be added for the [Strategic Finance Core Unit (SF-001)], if this executive proposal passes." or meant for me to add the link. I'll add it, but feel free to remove if you think that's better.

Reply:
> Just double checking that this isn't a typo: Contributor 2's stream start date is also 2022 in the spreadsheet

This was a typo - ARGH